doc/protocol.md

```diff
@@ -289,9 +289,10 @@ is no longer supported.
 Additional "open" command options are needed to open a channel of this
 type:
 
- * "bus": The DBus bus to connect to either "session" or "system",
-   defaults to "system" if not present. If set to "internal" then this
-   channel will communicate with the internal bridge DBus connection.
+ * "bus": The DBus bus to connect to either "session", "system" or a dbus
+   supported address. Defaults to "system" if not present. If set to
+   "internal" then this channel will communicate with the internal bridge
+   DBus connection.
  * "name": A service name of the DBus service to communicate with. Set to
    null if "bus" is "internal".
```

Comment:
This line probably wants an update ... also null when a DBus supported address right?